src/commons/stores/user-store.js

```diff
@@ -0,0 +1,69 @@
+(function() {
+    "use strict";
+
+    angular
+        .module("TendrlModule")
+        .service("userStore", userStore);
+
+    /*@ngInject*/
+    function userStore($state, $q, userFactory) {
+        var store = this;
+
+        store.getUserList = function() {
+            var deffered;
+
+            deffered = $q.defer();
+            userFactory.getUserList()
+                .then(function(data) {
+                    deffered.resolve(data);
```

Comment:
@a2batic You should call setupUserListData here itself instead of calling from userController(no need of making it as public funtion. It should be private for userStore). Please refer - https://github.com/Tendrl/ui/blob/ui-legacy-code/src/commons/stores/task-store.js.